example/src/main/java/com/ncorti/slidetoact/example/SampleActivity.java

```diff
@@ -85,11 +88,13 @@ public void onClick(final View v) {
                     @Override
                     public void onSlideReset(SlideToActView view) {
                         slider.setSliderIcon(R.drawable.custom_icon);
+                        slider.setCompleteIcon(R.drawable.slidetoact_animated_ic_check);
                     }
```

Comment:
Let's remove this entirely. It's more confusing given that the `completeIcon` is visible only at the end.

Reply:
done